src/test/java/uk/gov/justice/digital/hmpps/csr/api/service/DetailServiceTest.kt

```diff
@@ -150,11 +247,12 @@ internal class DetailServiceTest {
             val details = listOf(getValidShiftDetail(86401L, 456L))
             every { sqlRepository.getDetails(from, to, quantumId) } returns details
             every { authenticationFacade.currentUsername } returns quantumId
+            every { sqlRepository.getDetailTemplates(emptyList()) } returns emptyList<DetailTemplate>()
 
             val returnValue = service.getStaffDetails(from, to, quantumId)
 
             verify { sqlRepository.getDetails(from, to, quantumId) }
-            confirmVerified(sqlRepository)
+            verify { sqlRepository.getDetailTemplates(emptyList()) }
```

Comment:
Its still useful to make both of these assertions. confirmVerified means "and nothing else happened"

Reply:
See comment https://github.com/ministryofjustice/csr-api/pull/22#discussion_r488587005